platform/database/tests/database_test.go

```diff
@@ -0,0 +1,281 @@
+package tests
+
+import (
+	"context"
+	"encoding/json"
+	"os"
+	"testing"
+
+	"github.com/GuiaBolso/darwin"
+	"github.com/fewlinesco/go-pkg/platform/database"
+)
+
+func TestProdDatabase(t *testing.T) {
+	defer func() {
+		if err := recover(); err != nil {
+			t.Fatalf("should not have panicked but got panic with %#v", err)
+		}
+	}()
+	cfgfile, err := os.Open("./testdata/databaseConfig.json")
+	if err != nil {
+		t.Fatalf("can't open databaseConfig file: %#v", err)
+	}
+
+	cfg := database.DefaultConfig
+
+	if err := json.NewDecoder(cfgfile).Decode(&cfg); err != nil {
+		t.Fatalf("can't parse file: %#v", err)
+	}
+
+	db, err := database.Connect(cfg)
+	if err != nil {
+		t.Fatalf("could not connect to DB: %#v, with config: %#v", err, cfg)
+	}
+	defer func() { db.Close() }()
+
+	err = database.Migrate(db, []darwin.Migration{
+		{
+			Version:     1,
+			Description: "Create test data table",
+			Script: `CREATE TABLE test_data(
+				id UUID PRIMARY KEY,
+				value VARCHAR(63)
```

Comment:
Having this column named `value` makes it kinda hard to reason about the data in this test, when I say:

"You are inserting a wrong value" am I talking about the object you're trying to insert or the value of the value column? I think this kind of confusion can be easily avoided by renaming this column to `email` or `code` or something else...